### ISSUE TITLE
Build Error

### ISSUE THREAD

**Issue by highwaywarrior0** [ZTzTopia/GTProxy#66]
```
C:\GTProxy\build>cmake` .. -DCMAKE_BUILD_TYPE=Debug
-- Building for: Visual Studio 17 2022
-- Selecting Windows SDK version 10.0.22000.0 to target Windows 10.0.19045.
-- The C compiler identification is MSVC 19.36.32535.0
-- The CXX compiler identification is MSVC 19.36.32535.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.36.32532/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.36.32532/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for fcntl
-- Looking for fcntl - not found
-- Looking for poll
-- Looking for poll - not found
-- Looking for getaddrinfo
-- Looking for getaddrinfo - not found
-- Looking for getnameinfo
-- Looking for getnameinfo - not found
-- Looking for gethostbyname_r
-- Looking for gethostbyname_r - not found
-- Looking for gethostbyaddr_r
-- Looking for gethostbyaddr_r - not found
-- Looking for inet_pton
-- Looking for inet_pton - not found
-- Looking for inet_ntop
-- Looking for inet_ntop - not found
-- Performing Test HAS_MSGHDR_FLAGS
-- Performing Test HAS_MSGHDR_FLAGS - Failed
-- Check size of socklen_t
-- Check size of socklen_t - failed
-- GTProxy version: 1.4.2
-- CMake-Conan: first find_package() found. Installing dependencies with Conan
-- CMake-Conan: Checking if a default profile exists
ERROR: Profile not found: default
-- CMake-Conan: The default profile doesn't exist, detecting it.
Found msvc 17
WARN: This profile is a guess of your environment, please check it.
WARN: The output of this command is not guaranteed to be stable and can change in future Conan versions.
WARN: Use your own profile files for stability.
Saving detected profile to C:\Users\qq\.conan2\profiles\default
Detected profile:
[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=14
compiler.runtime=dynamic
compiler.version=193
os=Windows

-- CMake-Conan: cmake_system_name=Windows
-- CMake-Conan: CMake compiler=MSVC
-- CMake-Conan: CMake cmpiler version=19.36.32535.0
-- CMake-Conan: [settings] compiler=msvc
-- CMake-Conan: [settings] compiler.version=193
-- CMake-Conan: Creating profile C:/GTProxy/build/conan_host_profile
-- CMake-Conan: Profile:
include(default)
[settings]
os=Windows
compiler=msvc
compiler.version=193
compiler.cppstd=20
[conf]
tools.cmake.cmaketoolchain:generator=Visual Studio 17 2022

-- CMake-Conan: Installing both Debug and Release
-- CMake-Conan: conan install C:/GTProxy -of=C:/GTProxy/build/conan -pr;C:/GTProxy/build/conan_host_profile;-s;build_type=Release;--build=missing;-g;CMakeDeps

======== Input profiles ========
Profile host:
[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=20
compiler.runtime=dynamic
compiler.runtime_type=Release
compiler.version=193
os=Windows
[conf]
tools.cmake.cmaketoolchain:generator=Visual Studio 17 2022

Profile build:
[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=14
compiler.runtime=dynamic
compiler.runtime_type=Release
compiler.version=193
os=Windows


======== Computing dependency graph ========
cpp-httplib/0.12.6: Not found in local cache, looking in remotes...
cpp-httplib/0.12.6: Checking remote: conancenter
cpp-httplib/0.12.6: Downloaded recipe revision 5a09d54eac15e6f979e24676a3efd74a
fmt/9.1.0: Not found in local cache, looking in remotes...
fmt/9.1.0: Checking remote: conancenter
fmt/9.1.0: Downloaded recipe revision 44302d39c5a4bf7de8a39adc50bb4568
libressl/3.5.3: Not found in local cache, looking in remotes...
libressl/3.5.3: Checking remote: conancenter
libressl/3.5.3: Downloaded recipe revision ca95bb7d1b2b78bad04f3d37528158a2
magic_enum/0.8.2: Not found in local cache, looking in remotes...
magic_enum/0.8.2: Checking remote: conancenter
magic_enum/0.8.2: Downloaded recipe revision b5498cdc3f30cf47a1eab005f5012f2c
nlohmann_json/3.11.2: Not found in local cache, looking in remotes...
nlohmann_json/3.11.2: Checking remote: conancenter
nlohmann_json/3.11.2: Downloaded recipe revision a35423bb6e1eb8f931423557e282c7ed
pcg-cpp/cci.20210406: Not found in local cache, looking in remotes...
pcg-cpp/cci.20210406: Checking remote: conancenter
pcg-cpp/cci.20210406: Downloaded recipe revision 1891d8a4d379a1d2fe13f9a554671bdc
spdlog/1.11.0: Not found in local cache, looking in remotes...
spdlog/1.11.0: Checking remote: conancenter
spdlog/1.11.0: Downloaded recipe revision 8b4bd1da26e0ec97698369d6224d93bf
Graph root
    conanfile.py: C:/GTProxy\conanfile.py
Requirements
    cpp-httplib/0.12.6#5a09d54eac15e6f979e24676a3efd74a - Downloaded (conancenter)
    fmt/9.1.0#44302d39c5a4bf7de8a39adc50bb4568 - Downloaded (conancenter)
    libressl/3.5.3#ca95bb7d1b2b78bad04f3d37528158a2 - Downloaded (conancenter)
    magic_enum/0.8.2#b5498cdc3f30cf47a1eab005f5012f2c - Downloaded (conancenter)
    nlohmann_json/3.11.2#a35423bb6e1eb8f931423557e282c7ed - Downloaded (conancenter)
    pcg-cpp/cci.20210406#1891d8a4d379a1d2fe13f9a554671bdc - Downloaded (conancenter)
    spdlog/1.11.0#8b4bd1da26e0ec97698369d6224d93bf - Downloaded (conancenter)
Resolved version ranges
    cpp-httplib/[~0.12]: cpp-httplib/0.12.6
    libressl/[~3.5]: libressl/3.5.3
    magic_enum/[~0.8]: magic_enum/0.8.2
    nlohmann_json/[~3.11]: nlohmann_json/3.11.2
    spdlog/[~1.11]: spdlog/1.11.0
ERROR: Version conflict: spdlog/1.11.0->fmt/10.0.0, ->fmt/9.1.0.
CMake Error at vendor/cmake-conan/conan_support.cmake:147 (message):
  Conan install failed='1'
Call Stack (most recent call first):
  vendor/cmake-conan/conan_support.cmake:178 (conan_install)
  src/CMakeLists.txt:52 (find_package)


CMake Error at src/CMakeLists.txt:52 (find_package):
  By not providing "Findfmt.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "fmt", but
  CMake did not find one.

  Could not find a package configuration file provided by "fmt" with any of
  the following names:

    fmtConfig.cmake
    fmt-config.cmake

  Add the installation prefix of "fmt" to CMAKE_PREFIX_PATH or set "fmt_DIR"
  to a directory containing one of the above files.  If "fmt" provides a
  separate development package or SDK, be sure it has been installed.


-- Configuring incomplete, errors occurred!
```

why

**Comment by badewen** [comment]
fixed in #72

**Comment by ZTzTopia** [comment]
Should have been fixed in this commit 49ca26505ed351cff3555371e5bdc7ee537d79b7
